examples/jsm/renderers/webgl/nodes/WebGLNodes.js

```diff
@@ -47,3 +48,55 @@ Material.prototype.onBeforeRender = function ( renderer, scene, camera, geometry
 	}
 
 };
+
+WebGLRenderer.prototype.compute = async function ( ...computeNodes ) {
```

Comment:
Maybe we can add some caching system here... Because currently `material` is recreated per every `.compute()` call, so any compute node must be rebuilt.